docs/reference/mount-write-mode-semantics.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Mount Write Mode Semantics
+description: Everest mount file system semantics, limitations and and general info.
+grand_parent: Features
+parent: Mount
+---
+# Mount Write Mode Semantics
+
+{: .d-inline-block }
+experimental
+{: .label .label-red }
+
+## Consistency Model
+
+### File System Consistency
+Everest mount provides a strong read-after-write consistency model. 
```

Comment:
```suggestion
Everest mount provides a strong read-after-write consistency model within a single mount point. 
```

just to clarify that this doesn't happen if we mount the same path at different locations